src/meta/meta_ckpt.c

```diff
@@ -1028,21 +1042,26 @@ __wt_metadata_update_base_write_gen(WT_SESSION_IMPL *session, const char *config
 }
 
 /*
- * __wt_metadata_init_base_write_gen --
- *     Initialize the connection's base write generation.
+ * __wt_metadata_load_prior_state --
+ *     Initialize the connection's base write generation and most recent checkpoint time
```

Comment:
Missing a period here.

Reply:
Fixed in [a30614b](https://github.com/wiredtiger/wiredtiger/pull/8245/commits/a30614b0b9f9c9da6a61ed0e7fb4575c41456224)